### PR TITLE
Added profile page styling

### PR DESCRIPTION
**Pull request by sthewissen**
- Styled the profile page according to #21. 
- Added support for multiline Award Categories
- Tweaked font sizes on contribution list page
- Added support for MVP number on the page, not in the API yet

### CODE REVIEW

File: MVP/MVP/MVP/ViewModels/MyProfileViewModel.cs [hunk 1]
```diff
@@ -263,6 +265,23 @@ public string ErrorMessage
                 OnPropertyChanged("HasError");
             }
         }
+
+
+
+        public bool IsLoading
```
Comment (jamesmontemagno) [inline MVP/MVP/MVP/ViewModels/MyProfileViewModel.cs]: Isn't this the same as IsBusy, which is already on there?

**Review by jamesmontemagno — CHANGES_REQUESTED** [review]

File: MVP/MVP/MVP/ViewModels/MyProfileViewModel.cs (hunk 1 above)
Comment (sthewissen) [reply MVP/MVP/MVP/ViewModels/MyProfileViewModel.cs]: Good point.

File: MVP/MVP/MVP.iOS/AppDelegate.cs [hunk 2]
```diff
@@ -24,29 +24,37 @@ public partial class AppDelegate : global::Xamarin.Forms.Platform.iOS.FormsAppli
...
-			global::Xamarin.Forms.Forms.Init();
+            global::Xamarin.Forms.Forms.Init();
+
+            ImageCircle.Forms.Plugin.iOS.ImageCircleRenderer.Init();
 
-			ImageCircle.Forms.Plugin.iOS.ImageCircleRenderer.Init();
-            //Xamarin.FormsMaps.Init();
             LoadApplication(new App());
 
+            foreach (NSString family in UIFont.FamilyNames)
```
Comment (jamesmontemagno) [inline MVP/MVP/MVP.iOS/AppDelegate.cs]: Can we remove this?

**Review by jamesmontemagno — CHANGES_REQUESTED** [review]

File: MVP/MVP/MVP.iOS/AppDelegate.cs (hunk 2 above)
Comment (sthewissen) [reply MVP/MVP/MVP.iOS/AppDelegate.cs]: I removed it, had definitely served its use. Also removed some statusbar code that became obsolete due to the fix in navbar/statusbar colors.

**Review by jamesmontemagno — APPROVED** [review]

**Comment by jamesmontemagno** [comment]
Anything else or good to go?

**Comment by sthewissen** [comment]
I think that's it for the profile page.

**Pull request merged**